git/git.go

```diff
@@ -80,6 +96,19 @@ func ResolveRef(ref string) (*Ref, error) {
 	return fullref, nil
 }
 
+func ResolveRefs(refnames []string) ([]*Ref, error) {
+	refs := make([]*Ref, len(refnames))
+	for i, name := range refnames {
+		ref, err := ResolveRef(name)
+		if err != nil {
+			return refs, err
+		}
+
+		refs[i] = ref
+	}
+	return refs, nil
+}
```

Comment:
This sucks, because `ResolveRef()` runs `simpleExec()`. You can pass multiple refs though:

```bash
$ git rev-parse master push-extraction push-optimizations --symbolic-full-name master push-extraction push-optimizations
2cf082fa14a738f73db6d172eba800ca118e6d97
ce738288364c00185a51c6fd2c9ce42d4ddec155
8aae3553fa708503571fa8fea90ac54a82b7b31b
refs/heads/master
refs/heads/push-extraction
refs/heads/push-optimizations
```